docs/RELEASE.md

```diff
@@ -39,16 +39,18 @@ Publishing involves creating a release tag and creating a new *Release* on GitHu
 1. Commit the changes and push to remote repository to create a pull request.
 
 	```
-	git commit -a -s -m "Prepare <version> release"
-    git commit --amend --signoff
-	git push <FORK> <FEATURE-BRANCH>
+	git checkout -b release-<NEW VERSION>
+	git commit -a -s -m "Prepare <NEW VERSION> release" --signoff
```

Comment:
`-s` == `--signoff` so we just need one of them

Reply:
oh cool, I thought we weren't signing off due to the --amend